test/scripts/e2e-cp.sh

```diff
@@ -136,9 +131,9 @@ go test -test.timeout 180m -v . \
         -ginkgo.timeout 3h \
         -ginkgo.flake-attempts ${FLAKE_ATTEMPTS:-2} \
         -ginkgo.skip="${SKIPPED_TESTS}" \
+        -ginkgo.junit-report=${E2E_REPORT_DIR}/junit_${E2E_REPORT_PREFIX}report.xml \
         -provider skeleton \
         -kubeconfig ${KUBECONFIG} \
         ${NUM_NODES:+"--num-nodes=${NUM_NODES}"} \
-        ${E2E_REPORT_DIR:+"--report-dir=${E2E_REPORT_DIR}"} \
-        ${E2E_REPORT_PREFIX:+"--report-prefix=${E2E_REPORT_PREFIX}"}
```

Comment:
Opportunistic request. While we are here, could you please change line 7 of this script to something more useable? Specifically:
```
export KUBECONFIG=${KUBECONFIG:-${HOME}/ovn.conf}
```
I really dislike how that is hardcoded like that. :(

Reply:
@flavio-fernandes nice to see you again after long time :) @tssurya, this may be helpful when e2e's are run via make targets on a cluster.